packages/angular/cli/commands/new.json

```diff
@@ -33,6 +33,11 @@
       "default": false,
       "aliases": ["v"],
       "description": "Adds more details to output logging."
+    },
+    "non-interactive": {
```

Comment:
No negative flags please. Make the default to `true`.

Reply:
Done.